docs/src/tool/index.md

```diff
@@ -226,26 +232,28 @@ may be used in any JavaScript project like any other JavaScript file and library
 
 # {#ref-usage-devnet} `reach devnet`
 
-You can run a private Reach devnet by executing
+You can run a private Reach devnet by executing the following command:
 
 ```cmd
 $ reach devnet
 ```
 
-`reach devnet` supports the following options:
+:::note
+If running `reach devnet`, it is recommended to permanently set `REACH_CONNECTOR_MODE` to the desired consensus network. 
+If you did not set this when you originally ran `reach config`, then you can run `reach devnet` with the argument `REACH_CONNECTOR_MODE=[OPTION]` as in the following command.
+:::
 
-+ `--await-background` --- Run in background and await availability.
-+ The mandatory environment variable `REACH_CONNECTOR_MODE` specifies which devnet to run.
-The options are:
+```cmd
+$ REACH_CONNECTOR_MODE=ALGO ./reach devnet
+```
 
-+ `ETH`, which runs an Ethereum devnet on `localhost:8545`
-+ `ALGO`, which runs an Algorand devnet on `localhost:4180` and an Algorand indexer on `localhost:8980`
+For more information on devnet options, refer to [Algorand](##ref-network-algo), [Conflux](##ref-network-cfx), or [Ethereum](##ref-network-eth).
```

Comment:
I'd link to the network page itself, not the individual options. That way we don't forget choices as we add more